relayer/src/supervisor/spawn.rs

```diff
@@ -39,24 +40,32 @@ pub struct SpawnContext<'a> {
     config: &'a RwArc<Config>,
     registry: &'a mut Registry,
     workers: &'a mut WorkerMap,
+    client_state_filter: &'a mut FilterPolicy,
```

Comment:
FYI @romac I had to propagate the mutable `client_state_filter` from the supervisor into the SpawnContext. Did not see a simpler approach than this.